.github/workflows/format-code.yml

```diff
@@ -0,0 +1,49 @@
+name: "Format C++ code"
```

Comment:
What does this workflow do? I _think_ it lets us comment using either of those two format strings on a pull request and have the CI make a formatting commit, is that right?

Reply:
Yes, it does exactly that. Something similar is already in use for the C# track and for Markdown and JSON files in the V3 repo.

For example in PR exercism/v3#1788 you can see the one for Markdown and Json in action.